docs/modules/ROOT/pages/native-memory.adoc

```diff
@@ -0,0 +1,51 @@
+= Configuring the Native Memory Store
+:description: The Native Memory Store is a pluggable memory manager. It allows you to set up different configurations, or stores, for your application’s data structures on a single JVM. These memory stores form part of a common access layer which scales up to use a configured amount of memory on the JVM.
+
+{description}
+
+As well as providing more predictable application scaling, using a Native Memory Store boosts performance, reduces latency, and minimizes garbage collection pauses. 
+
+== Configuring a Native Memory Store
+
+Below are the configuration options for the Native Memory Store.
+
+[cols="25%m,75%a"]
+|===
+|Field|Description
+
+|`allocatorType`
+| Specifies the mechanism for allocating memory; either XXXX or XXXX.
```

Comment:
For this line, you need to add in the name of the two allocator types. It looks like the first is `POOLED` but I'm not sure about the second one.